core/include/kernel/pm.h

```diff
@@ -169,8 +195,23 @@ static inline void register_pm_core_service_cb(pm_callback callback,
 					       void *handle, const char *name)
 {
 	register_pm_cb(&PM_CALLBACK_HANDLE_INITIALIZER(callback, handle,
-						PM_CB_ORDER_CORE_SERVICE,
-						name));
+						       PM_CB_ORDER_CORE_SERVICE,
+						       name));
+}
+
+/*
+ * Unregister a core service callback from generic suspend/resume sequences
+ *
+ * @callback: Registered callback function
+ * @handle: Private handle argument that was registered for the callback
+ */
+static inline void unregister_pm_core_service_cb(pm_callback callback,
+						 void *handle)
+{
+	enum pm_callback_order order = PM_CB_ORDER_CORE_SERVICE;
```

Comment:
Local variable not needed

Reply:
It was to keep alignment with parentheses without exceeding the character per lign limit. Do you suggest I change it nevertheless?

Reply:
OK then it's fine as it is. In fact `PM_CALLBACK_HANDLE_INITIALIZER` is too long, it should have been called `PM_CB_HANDLE_INITIALIZER`.

Reply:
By the way, we could also shorten `struct pm_callback_handle` to `struct pm_cb_handle` :)